report/verification/mms/simple_transient.md

```diff
@@ -0,0 +1,312 @@
+---
+jupytext:
+  formats: ipynb,md:myst
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.16.2
+kernelspec:
+  display_name: vv-festim-report-env
+  language: python
+  name: python3
+---
+
+# Simple transient diffusion case
+
+```{tags} 2D, MMS, transient
+```
+
+This is a simple transient MMS example.
+We will only consider diffusion of hydrogen in a unit square domain $\Omega$ at steady state with an homogeneous diffusion coefficient $D$.
+Moreover, a Dirichlet boundary condition will be assumed on the boundaries $\partial \Omega $.
+
+The problem is therefore:
+
+$$
+\begin{align}
+    &\nabla \cdot (D \ \nabla{c}) - \frac{\partial c}{\partial t} = -S  \quad \text{on }  \Omega  \\
+    & c = c_0 \quad \text{on }  \partial \Omega
+\end{align}
+$$(problem_simple_transient)
+
+The exact solution for mobile concentration is:
+
+$$
+\begin{equation}
+    c_\mathrm{exact} = 1 + 2 x^2 + 3 y^2 t + 2t
```

Comment:
```suggestion
    c_\mathrm{exact} = 1 + 2 x^2 + 3 y^2 + 2t
```

I think this is a typo

Reply:
no, this is the function I used. because of that `t` we get the `3y^2` and `-6tD` terms in the solution for `S`.